docs/src/layouts/partials/Header.vue

```diff
@@ -5,8 +5,6 @@
       Nav.flex-fit
       SearchForm
       nav.header-actions.flex
-        a(aria-label="Twitter" href="//twitter.com/Ulixee" rel="noopener noreferrer" target="_blank" title="Follow us on Twitter")
-          twitter-logo
         a(aria-label="Discord" href="//discord.gg/ydycFGU" rel="noopener noreferrer" class="hide-for-small" target="_blank" title="Join our discord")
```

Comment:
Is this the same discord as before?

Reply:
Ie, SecretAgent one?

Reply:
Good catch. I created a Ulixee server and updated link.